microsoft-365/business/manage-windows-devices.md

```diff
@@ -32,19 +32,36 @@ To set up your organization's domain-joined devices to benefit from the capabili
+
+Then, to register existing domain-joined Windows 10 devices as Hybrid Azure AD Joined, follow the step by step instructions in the [Tutorial: Configure hybrid Azure Active Directory join for managed domains](https://docs.microsoft.com/azure/active-directory/devices/hybrid-azuread-join-managed-domains#configure-hybrid-azure-ad-join). This will hybrid-enable your existing on-premises Active Directory joined Windows 10 computers and make them cloud ready.
     
-3. Register existing domain-joined Windows 10 devices to be hybrid Azure AD Joined and enroll them for mobile device management by Intune (Microsoft 365 Business):
-    
-4. Follow the step by step instructions in [How to configure hybrid Azure Active Directory joined devices](https://go.microsoft.com/fwlink/p/?linkid=872870). This will enable the synchronization of your on-premises Active Directory joined Windows 10 computers and make them cloud ready.
-    
-5. In order to enroll a Windows 10 device for mobile device management, see [Enroll a Windows 10 device with Intune by using a Group Policy](https://go.microsoft.com/fwlink/p/?linkid=872871) for instructions. You can set the Group Policy at a local computer level or for bulk operations, you can create this group policy setting on your domain controller server.
\ No newline at end of file
+4. **Enable automatic enrollment for Windows 10**: To automatically enroll Windows 10 devices for mobile device management in Intune, see [Enroll a Windows 10 device automatically using Group Policy](https://docs.microsoft.com/windows/client-management/mdm/enroll-a-windows-10-device-automatically-using-group-policy) for instructions. You can set the Group Policy at a local computer level, or for bulk operations, you can create this group policy setting on your Domain Controller using the Group Policy Management Console and ADMX templates.
+
+5. **Configure Seamless SSO**: Seamless Single Sign-On will automatically sign in users to their Microsoft 365 cloud resources when they are on their corporate desktops. Simply deploy one of the two Group Policy options described in [Azure Active Directory Seamless Single Sign-On: Quick start](https://docs.microsoft.com/azure/active-directory/hybrid/how-to-connect-sso-quick-start#step-2-enable-the-feature). The "Group policy" option does not allow users to change their settings, while the "Group policy preference" option sets the values but also leaves them user-configurable.
+
+6. **Setup Windows Hello for Business**: Windows Hello for Business replaces passwords with strong two-factor authentication (2FA) for signing into a local computer. One factor is an asymmetric key pair, and the other is a PIN or other local gesture such as fingerprint or facial recognition if your device supports it. It is recommended to replace passwords with 2FA and Windows Hello for Business where possible.
```

Comment:
```suggestion
6. **Set up Windows Hello for Business**: Windows Hello for Business replaces passwords with strong two-factor authentication (2FA) for signing into a local computer. One factor is an asymmetric key pair, and the other is a PIN or other local gesture such as fingerprint or facial recognition if your device supports it. We recommended that you replace passwords with 2FA and Windows Hello for Business where possible.
```

Reply:
Another case of the past tense "recommended" in here: "We recommended that you..." --my bad